### PR TITLE
Main menu

### PR DESCRIPTION
**Pull request by drcaramelsyrup**
Main menu features: continue button and new game button.

### CODE REVIEW

File: src/objects/Clickable.js [hunk 1]
```diff
@@ -31,6 +31,20 @@ function click() {
...
+  //because it is checking the id of the item and not the actual item
+  for (let i = 0; i < items.length; i++) {
+    if (items[i].id === this.id){
+      remove_item = i;
+    }
+  }
+  if (remove_item > -1) {
+    items.splice(remove_item, 1);
+  } else {
+    console.log('unexpected error, please check');
```
Comment (drcaramelsyrup) [inline src/objects/Clickable.js:45]: If this is meant to be more of an assert function, I think you can try using [console.assert()](https://developer.mozilla.org/en-US/docs/Web/API/console/assert).

File: src/states/Preloader.js [hunk 2]
```diff
@@ -18,10 +19,12 @@ function showSplashScreen (game) {
 exports.preload = function (game) {
   showSplashScreen(game);
   game.load.pack('game', null, assets);
+  game.load.pack('main_menu', null, assets);
   game.load.pack('ui', null, assets);
   game.load.pack('conversations', null, assets);
   game.load.pack('sounds', null, assets);
   game.load.pack('logic_battle', null, assets);
+  //game.load.script('webfont', '//ajax.googleapis.com/ajax/libs/webfont/1.4.7/webfont.js');
```
Comment (drcaramelsyrup) [inline src/states/Preloader.js:27]: Extra comment?

File: src/states/Preloader.js [hunk 3]
```diff
@@ -34,6 +37,8 @@ exports.create = function (game) {
   // between preload and create functions FOR EACH STATE
   game.slickUI = game.plugins.add(Phaser.Plugin.SlickUI);
 
-  game.state.start('Game');
+  //TODO delete this shit
```
Comment (drcaramelsyrup) [inline src/states/Preloader.js:40]: delete this shit

File: static/index.html [hunk 4]
```diff
@@ -16,7 +16,7 @@
     <!-- For Google WebFonts -->
     <script type="text/javascript">
       WebFontConfig = {
-        google: { families: [ 'Open Sans', 'Goudy Bookletter 1911' ] }
+        google: { families: [ 'Open Sans', 'Goudy Bookletter 1911', 'Exo', 'Cinzel Decorative', "Megrim" ] }
```
Comment (drcaramelsyrup) [inline static/index.html:19]: Ooh!

**Review by drcaramelsyrup — COMMENTED** [review]
Cool! Could you just get rid of the couple comments and we're good to merge :)

**Comment by thequeenofspades** [comment]
Closing in favor of #22

**Pull request closed without merging**